README.md

```diff
@@ -1,2 +1,307 @@
+---
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: vm-client
+spec:
+  running: true
+  template:
+    spec:
+      domain:
+        devices:
+          disks:
+            - name: containerdisk
+              disk:
+                bus: virtio
+            - name: cloudinitdisk
+              disk:
+                bus: virtio
+          interfaces:
+          - name: default
+            masquerade: {}
+          - name: flatl2-overlay
+            bridge: {}
+        machine:
+          type: ""
+        resources:
+          requests:
+            memory: 1024M
+      networks:
+      - name: default
+        pod: {}
+      - name: flatl2-overlay
+        multus:
+          networkName: flatl2
+      terminationGracePeriodSeconds: 0
+      volumes:
+        - name: containerdisk
+          containerDisk:
+            image: quay.io/kubevirt/fedora-with-test-tooling-container-disk:v1.0.0
+        - name: cloudinitdisk
+          cloudInitNoCloud:
+            networkData: |
+              version: 2
+              ethernets:
+                eth1:
+                  dhcp4: true
+            userData: |-
+              #cloud-config
+              password: fedora
+              chpasswd: { expire: False }
+EOF
+```
+
+The yaml provided above will create two VMs, each with a network interface
+connected to the cluster default network (granting access to Kubernetes services
+and access to the internet), plus a secondary interface connected to the overlay
+defined in [the network attachment definition](#the-overlay-definition)
+presented above.
+
+Once both VMs have booted up, we can check their IP addresses, and confirm they
+can use the overlay to communicate:
+
+```bash
+# check the IP address of the `server` VM
+oc get vmi vm-server -ojsonpath="{@.status.interfaces}" | jq
+[
+  {
+    "infoSource": "domain, guest-agent",
+    "interfaceName": "eth0",
+    "ipAddress": "10.135.1.154",
+    "ipAddresses": [
+      "10.135.1.154"
+    ],
+    "mac": "02:2e:2f:00:00:0e",
+    "name": "default",
+    "queueCount": 1
+  },
+  {
+    "infoSource": "domain, guest-agent, multus-status",
+    "interfaceName": "eth1",
+    "ipAddress": "192.168.123.5",
+    "ipAddresses": [
+      "192.168.123.5",
+      "fe80::2e:2fff:fe00:f"
+    ],
+    "mac": "02:2e:2f:00:00:0f",
+    "name": "flatl2-overlay",
+    "queueCount": 1
+  }
+]
+
+# connect to the `client` VM over the serial console
+# user / password are both `fedora`
+virtctl console vm-client
+Successfully connected to vm-client console. The escape sequence is ^]
+
+vm-client login: fedora
+Password:
+[fedora@vm-client ~]$ ping 192.168.123.5
+ping 192.168.123.5
+PING 192.168.123.5 (192.168.123.5) 56(84) bytes of data.
+64 bytes from 192.168.123.5: icmp_seq=1 ttl=64 time=3.54 ms
+64 bytes from 192.168.123.5: icmp_seq=2 ttl=64 time=1.72 ms
+64 bytes from 192.168.123.5: icmp_seq=3 ttl=64 time=1.99 ms
+64 bytes from 192.168.123.5: icmp_seq=4 ttl=64 time=1.56 ms
+```
```

Comment:
A hint how to end the console might be nice to new users.

Reply:
I think that's past the goal of this post.

Reply:
fair point